### ISSUE TITLE
error compiling framework 3-2-1 in delphi seattle

### ISSUE THREAD

**Issue by fniero** [danieleteti/delphimvcframework#465]
Hello, 
for first time I'm trying to use the framework (3-2-1) but compiler (seattle) found the below errors
Someone can help me?
Thanks
Francesco

[dcc32 Error] MVCFramework.pas(3221): E2003 Undeclared identifier: 'AddPair'
[dcc32 Error] MVCFramework.pas(3233): E2003 Undeclared identifier: 'AddPair'

[dcc32 Error] MVCFramework.ActiveRecord.pas(2271): E2089 Invalid typecast
[dcc32 Error] MVCFramework.ActiveRecord.pas(2283): E2089 Invalid typecast
[dcc32 Error] MVCFramework.ActiveRecord.pas(2295): E2089 Invalid typecast

[dcc32 Error] MVCFramework.JSONRPC.Client.pas(160): E2003 Undeclared identifier: 'ResponseTimeout'

[dcc32 Fatal Error] DMVC.Expert.CodeGen.Templates.pas(35): F1026 File not found: 'dmvcframework.inc'

**Comment by danieleteti** [comment]
These are methods introduced in the recent versions. I don't have 10.0 Seattle anymore, can you change the code using IFDEFs?

All the needed DEFINES are already defined. You can use 

```{$IFDEF BERLINORBETTER}
//current code
{$ELSE}
//the code which works for you
{$ENDIF}
```

**Comment by fniero** [comment]
Sorry, but I don't know which is "the code which works for you", I don't know the framework source, if there is a package "d100" I presume it was tested, it's hard  beginning with a new framework and debug the source code
Thanks

**Comment by danieleteti** [comment]
@fniero if you are available, I should have identified which would be the problematic lines. Can I send to you the updated unit to test? If so, please, use the repo version instead of the release

**Comment by fniero** [comment]
I'm available but I have some questions:
- in your last webinar you told there is a 'manager' for every delphi version, who is for seattle?
- for every new framework version I'll have problems in souce code?  
I'm sorry but I don't know difference between repo and release version
thanks

**Comment by allanwolski** [comment]
Same issue here with Delphi 10.1 Berlin.
Using the repo version works.

Could you compile a new stable version with these fixes please?

**Comment by danieleteti** [comment]
Thank you for your proactive feedback. I'll do a new release ASAP. Can you run unit tests?

**Comment by allanwolski** [comment]
I don't have unit tests implemented yet.

**Comment by joaoduarte19** [comment]
This incompatibility has already been fixed in commit 140b70ba33c2d0fc5bb9122a27841714751e2bb3
